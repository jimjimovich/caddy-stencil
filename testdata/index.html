{
"title": "test json"
}
<p>Some HTML</p>