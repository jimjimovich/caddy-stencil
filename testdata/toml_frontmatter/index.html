+++
footer = "Finally, a footer from <strong>TOML</strong>"
title = "Title from TOML"
+++

<p>
    Just some <strong>HTML</strong>.
</p>