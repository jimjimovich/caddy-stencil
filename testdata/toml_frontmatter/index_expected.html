<!DOCTYPE html>
<html>
    <head>
        <title>Title from TOML</title>
    </head>
    <body>
        
<p>
    Just some <strong>HTML</strong>.
</p>

        <p>Then some HTML from the template</p>
        Finally, a footer from <strong>TOML</strong>
    </body>
</html>